infra/discovery/src/lib/search.js

```diff
@@ -353,13 +437,10 @@ class Listing {
       minPrice: minPrice || 0,
       totalNumberOfListings: searchResponse.hits.total
     }
-
-    if (idsOnly) {
-      const listingIds = listings.map(listing => listing.id)
-      return { listingIds, stats }
-    } else {
-      return { listings, stats }
-    }
+    // debug
+    // console.log('listings - ', listings)
+    const listingIds = listings.map(listing => listing.id)
+    return { listingIds, listings, stats }
```

Comment:
This is a question I'd like @nick or @DanielVF to weigh in on. The original [code ](https://github.com/OriginProtocol/origin/pull/2873/files/d622f02059b113a6849e9b5d6e862d52a864549b#diff-e8a9317173ba9fe5906263f858dbf137L15-27)only returned `listingIds` (hardcoded ) so I added `listings` to get the price and currency information. But the subsequent code didn't complain when it received listings instead of listingIds. So I feel like the answer is yes, we could just return listings. I'll wait till they weigh in though.

Reply:
I'm good with returning just listings internally here.

Reply:
Yea, ultimately the dapp only queries for listing IDs but no harm in returning listings here.

Reply:
I'll remove the listingIds then.